### ISSUE TITLE
Gang war mafia update


### ISSUE THREAD

**Issue by Sahi2l** [skylicht-lab/skylicht-engine#138]
Hello sir 

*Very hard I got your gmail and you are not replying us on other social media platforms. If you really see our gmail then please Reply* 

This game is not even on play store(GANG WAR MAFIA) .  Nor are any of its servers running.  this game is completely closed

Can you tell me when the new format of this game will be released on play store

This is my only favorite FPS game and the only favorite of many people with me

*I am attaching some sacreenshot together so that you will understand.  what i want to say*

Tranclet in Vietnamese

Xin chào ông

 * Rất khó, tôi đã nhận được gmail của bạn và bạn không trả lời chúng tôi trên các nền tảng mạng xã hội khác.  Nếu bạn thực sự thấy gmail của chúng tôi thì hãy Trả lời *

 Trò chơi này thậm chí không có trên cửa hàng chơi (GANG WAR MAFIA).  Cũng không có bất kỳ máy chủ nào của nó đang chạy.  trò chơi này đã hoàn toàn đóng cửa

 Bạn có thể cho tôi biết khi nào định dạng mới của trò chơi này sẽ được phát hành trên Play Store không

 Đây là trò chơi FPS yêu thích duy nhất của và là trò chơi yêu thích duy nhất của nhiều người cùng tôi

*Tôi đính kèm một số sacreenshot với nhau để bạn hiểu. những gì tôi muốn nói*

**Comment by ducphamhong** [comment]
Hi Sahi2l 

I am very happy because GWM is your game favorite. Although so a long time since 2018, Our team has not had any updates on it.

Since 2013, I have written the game engine that is named Skylicht. And GWM is a product that is based on Skylicht Engine. I have a contract with VTC Academy to publish the GWM on PlayStore. But in 2019 the VTCA company is closed and sold to the new owner. So GWM doesn't have any investment to continue developing. 

Instead, I continue to grow Skylicht Engine to the Publish Open Source Engine at Github. That target, I want to improve the quality of the new game in feature. And we passed about 70% complete this Engine.

And I have kept the server alive from 2019 to the end of 2021. I stopped all game servers on 1/2022. I will focus to finish the Engine soon on 6/2022 and after that, I will write new a shooter game on our technology. Our team is also ready to do that.

Difficult to say when we continue releasing the new version GWM. Maybe that will be a new game but it is like GWM. Yes, it is FPS Online and Co-op features... We must have a demo of the game, And I am sure I can find the investment after the early demo version. Or I also can choose the way to release an NFT Game based on Crypto.

If you follow this Github link, you will see I still working every day to make a new product.

Thanks for playing GWM.
Regards,
Pham Hong Duc
Director of Skylicht LTD

**Comment by Sahi2l** [comment]
I can't tell.  How happy I am today because I got a reply from your side.
I just told all my friends.  They are all very happy, but everyone has a
question in their mind.  If the company isn't there, will the game be new
too.  Wouldn't there even be graphics.  Won't this game be like the old game
Is the game likely to be completely different

**Comment by ducphamhong** [comment]
Yes, but because the graphics of GWM is old. And we can't re-use the game, that was sold to VTCA. It will have some legal problems.
But overall the new will be like GWM.

**Comment by Josean1221** [comment]
Hola solo un pregunta dónde puedo dar seguimiento a sus proyectos GWM es grandioso juego lamentablemente lo cerraron espero que cuando salga su nuevo videojuego me pudieran invitar a una beta

**Comment by CR44FT** [comment]
estou muito feliz com essa noticia, amo gang war mafia, e meu canal vive dele kkk então ter noticias sobre nosso joguinho, me deixa muito feliz.

**Comment by Sahi2l** [comment]
I did not reply to you for so many days, which is also one of the reasons.
I have tried many fps games.  But I didn't like any.  In the end, I have
taken this decision after giving up.  I'll wait for your game.  Till then i
will not play any other game and i have a lot of faith in you that you will
make this game(new game) engine even better than gang war mafia
The graphics you made in 2013 were great graphics at that time.  But now to
make 2023 high graphics in such a way that the player is attracted or else
the players will not come.
In the end, wishing you all the very best for the new game and I hope so.
You will also get a good sponsor.



Tranclet in Vietnamese


Tôi đã không trả lời cho bạn trong rất nhiều ngày, đó cũng là một trong
những lý do. Tôi đã thử nhiều trò chơi fps. Nhưng tôi không thích cái nào.
Cuối cùng, tôi đã đưa ra quyết định này sau khi từ bỏ. Tôi sẽ đợi trò chơi
của bạn. Cho đến lúc đó tôi sẽ không chơi bất kỳ trò chơi nào khác và tôi
rất tin tưởng vào bạn rằng bạn sẽ làm cho trò chơi (trò chơi mới) này thậm
chí còn tốt hơn cả băng đảng mafia

 Đồ họa bạn thực hiện vào năm 2013 là đồ họa tuyệt vời vào thời điểm đó.
Nhưng bây giờ để làm cho 2023 đồ họa cao theo cách mà người chơi bị thu
hút, nếu không người chơi sẽ không đến.

 Cuối cùng, chúc các bạn sẽ có được những điều tốt đẹp nhất cho trò chơi
mới và tôi cũng hy vọng như vậy. Bạn cũng sẽ nhận được một nhà tài trợ tốt.

**Comment by Donmartins7** [comment]
Queria ter notícias do gang war mafia, desenvolvedores podiam se pronúncia sobre o jogo?

**Comment by ZINPie** [comment]
> Hi Sahi2l 
> 
> I am very happy because GWM is your game favorite. Although so a long time since 2018, Our team has not had any updates on it.
> 
> Since 2013, I have written the game engine that is named Skylicht. And GWM is a product that is based on Skylicht Engine. I have a contract with VTC Academy to publish the GWM on PlayStore. But in 2019 the VTCA company is closed and sold to the new owner. So GWM doesn't have any investment to continue developing. 
> 
> Instead, I continue to grow Skylicht Engine to the Publish Open Source Engine at Github. That target, I want to improve the quality of the new game in feature. And we passed about 70% complete this Engine.
> 
> And I have kept the server alive from 2019 to the end of 2021. I stopped all game servers on 1/2022. I will focus to finish the Engine soon on 6/2022 and after that, I will write new a shooter game on our technology. Our team is also ready to do that.
> 
> Difficult to say when we continue releasing the new version GWM. Maybe that will be a new game but it is like GWM. Yes, it is FPS Online and Co-op features, Esport... We must have a demo of the game, And I am sure I can find the investment after the early demo version. Or I also can choose the way to release an NFT Game based on Blockchain.
> 
> If you follow this Github link, you will see I still working every day to make a new product.
> 
> Thanks for playing GWM.
> Regards,
> Pham Hong Duc
> Director of Skylicht LTD



After searching so far finally I found you through your facebook page link, Your work is quite extraordinary sir. GWM is a game that's so fun and not boring, there's always a worthy opponent, I'm very satisfied. 

Totally agree if you build GWM on the NFT blockchain version and create some NFT characters and weapons, that's cool. and I'm sure many people will invest in your NFT work sir. 

Currently there is an FPS game called Matr1x Fire NFT from China which will be released soon and they are doing a beta test, and their NFT sales are opensea today. their games work on PC, Mac, iOS, and Android.

That's all I can say, old GWM players may be waiting for new work and hope it will be there soon, ok sir, I wish you long and healthy life. God bless

**Comment by Sahi2l** [comment]
Hello sir hope you are fine, i don't know. Which of your mails is correct
but I have mentioned two or three mails together.
My only request to you is that I cannot wait any longer. My friends, can't
wait any longer so please give me a satisfactory answer as to when will the
update of GWM 2.0 come?

Xin chào ngài, hy vọng ngài ổn, tôi không biết. Thư nào của bạn đúng nhưng
tôi đã đề cập đến hai hoặc ba thư cùng nhau.
 Yêu cầu duy nhất của tôi đối với bạn là tôi không thể chờ đợi lâu hơn nữa.
Nóng lòng quá các bạn ơi cho mình xin câu trả lời thỏa đáng là khi nào thì
có bản cập nhật GWM 2.0?



   Thanks



   Sahil Bishnoi

**Comment by Sahi2l** [comment]
Hello sir hope you are fine, i don't know. Which of your mails is correct
but I have mentioned two or three mails together.
My only request to you is that I cannot wait any longer. My friends, can't
wait any longer so please give me a satisfactory answer as to when will the
update of GWM 2.0 come?

Xin chào ngài, hy vọng ngài ổn, tôi không biết. Thư nào của bạn đúng nhưng
tôi đã đề cập đến hai hoặc ba thư cùng nhau.
 Yêu cầu duy nhất của tôi đối với bạn là tôi không thể chờ đợi lâu hơn nữa.
Nóng lòng quá các bạn ơi cho mình xin câu trả lời thỏa đáng là khi nào thì
có bản cập nhật GWM 2.0?



   Thanks
